_metadata/repos.conf.d/alpine/provider.conf

```diff
@@ -1,12 +0,0 @@
-*v2*
-*v3.0/
-*v3.1/
-*v3.2/
-*v3.3/
-*v3.4/
-*v3.5/
-*/aarch64/
-*/armhf/
-*/ppc64le/
```

Comment:
While x86 and ARM are debatable, I still see no use case for PowerPC and mainframe archs. I think we should continue to exclude s390x and ppc641e.

Reply:
While I agree that not getting PowerPC and other odd architectures is a good idea I do think that we should pick a spot where we stop mirroring Alpine. I think that it would make sense to grab from 3.6 and up. Then we can prune as newer versions are released and deprecated.